roles/StackStorm.st2chatops/tasks/main.yml

```diff
@@ -198,6 +198,14 @@
   notify:
     - restart st2chatops
 
+- name: daemon-reload
+  become: yes
+  command: systemctl daemon-reload
```

Comment:
You can try to use Ansible's module https://docs.ansible.com/ansible/latest/modules/systemd_module.html for daemon-reload.

Simple command is discouraged. This task will also break ansible dry-run, idempotence and EL6 which we still support.